glib/src/error.rs

```diff
@@ -260,4 +261,18 @@ mod tests {
         let true_dynamic_res = result_from_gboolean!(ffi::GTRUE, "{} message", "Dynamic");
         assert!(true_dynamic_res.is_ok());
     }
+
+    #[test]
+    fn test_value() {
+        let e1 = Error::new(crate::FileError::Failed, "Failed");
+        // This creates a copy ...
```

Comment:
For this I have a solution planned for the release after this one FWIW so that we don't create copies unless needed.

Reply:
Don't forget to open an issue if not already done. :)